Tests/Nimble/objc/ObjcStringersTest.m

```diff
@@ -0,0 +1,24 @@
+@import XCTest;
+@import Nimble;
+
+@interface ObjcStringersTest : XCTestCase
+
+@end
+
+@implementation ObjcStringersTest
+
+- (void)testItCanStringifyArrays {
+    NSArray *array = @[@1, @2, @3];
+    NSString *result = NMBStringify(array);
+    
+    XCTAssert([result isEqualToString:@"(1, 2, 3)"], @"got <%@>, expected <(1, 2, 3)>", result);
```

Comment:
I don't see an issue for using Nimble expectations here. Please use it instead of XCTest to be more consistent